crate_universe/private/defaults.bzl

```diff
@@ -0,0 +1,14 @@
+"""A helper module defining generated information about crate_universe dependencies"""
```

Comment:
Should this file be deleted? It feels like this should only ever contain actual sha256 values?

Reply:
There's very likely a smarter way to do this but this was easiest so would like to defer additional improvements here to another PR unless you feel strongly.